docs/ARCHITECTURE.md

```diff
@@ -39,23 +39,23 @@ _Depends on: nothing_
 ### Data pipeline 📥➡️♨➡️📤
 
 1️⃣ **Inputs**
-- Local GTFS archive
+- A local GTFS archive or fully qualified URL to download distance a GTFS archive from
 - Command line arguments 
 
-2️⃣  **Validator loading**
-- locate all validators and load them
+2️⃣ **Validator loading**
+- Locate all validators and load them
 
-3️⃣  **Feed loading**
-- create `GtfsInput`
-  - read GTFS files
-  - create `GtfsTableContainer` from data
+3️⃣ **Feed loading**
+- Create `GtfsInput`
```

Comment:
"- Create `GtfsInput`" isn't going to be meaningful to most people reading this document. I would eliminate it and just have 3 bullet points in this section.